src/encord_active/cli/main.py

```diff
@@ -174,62 +197,134 @@ def import_local_project(
+            rich.print("Label glob specified without a transformer. Label glob is only allowed with a transformer.")
+            print(label_glob)
+            raise typer.Abort()
+
+        try:
+            label_result = file_glob(root, label_glob, images_only=False).matched
+        except NoFilesFoundError as e:
+            rich.print(
+                Panel(
+                    str(e),
+                    title=":fire: No Files Found From Label Glob :fire:",
+                    expand=False,
+                    style="yellow",
+                )
+            )
+            raise typer.Abort()
+
+    transformer_results: List[TransformerResult] = []
+    if transformer is not None:
+        transformers_found = load_transformers_from_module(transformer)
+        if not transformers_found:
+            pass
+        elif len(transformers_found) == 1:
+            transformer_results = transformers_found
+        else:
+            choices = list(map(lambda m: Choice(m, name=m.name), transformers_found))
+            transformer_results = i.checkbox(
+                message="Please choose which label adapters to use? Use [TAB] to select from the list.", choices=choices
+            ).execute()
+
     if dryrun:
         directories: Set[Path] = set()
         rich.print("[blue]Matches:[/blue]")
-        for file in glob_result.matched:
+        for file in data_result.matched:
             directories.add(file.parent)
             rich.print(f"[blue]{escape(file.as_posix())}[/blue]")
 
         print()
         rich.print("[yellow]Excluded:[/yellow]")
-        for file in glob_result.excluded:
+        for file in data_result.excluded:
             directories.add(file.parent)
             rich.print(f"[yellow]{escape(file.as_posix())}[/yellow]")
 
+        found_labels: Dict[str, Dict[str, int]] = {}  # <type, <class, label>>
+        for transformer_result in transformer_results:
+            labels = transformer_result.transformer.from_custom_labels(label_result, data_files=data_result.matched)
+            for label in labels:
+                label_type = type(label.label).__name__
+                label_name = label.label.class_
+                counter = found_labels.setdefault(label_type, {})
+                counter[label_name] = counter.get(label_name, 0) + 1
+
+            if not labels:
+                rich.print(f"[yellow]The transformer [blue]{transformer_result.name}[/blue] didn't return any labels")
+                continue
+
+            labels = sorted(labels, key=lambda l: l.abs_data_path)
+            rich.print(f"Labels identified by [blue]{transformer_result.name}[/blue]:")
+            current_file_name = None
+            for label in labels:
+                if label.abs_data_path != current_file_name:
+                    current_file_name = label.abs_data_path
+                    rich.print(f"[green]{current_file_name}[/green]")
+                rich.print(f"\t{label.label}")
```

Comment:
[nitpick] 💅

For better readability, can move the check if `labels` exists to above the loop that iterates them?
Also, can we merge the two loops iterating the labels?